src/main/java/appeng/fluids/helper/IAEFluidInventory.java

```diff
@@ -0,0 +1,11 @@
+
+package appeng.fluids.helper;
+
+
+import net.minecraftforge.fluids.capability.IFluidHandler;
+
+
+public interface IAEFluidInventory
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "IAEFluidInventory" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)